### PR TITLE
Fix the tooltips on mobile devices

### LINKED ISSUE #6382: Missing tooltips on the Contao Backend on touch devices
It seems like there is no touch support for the explanation tooltips below each backend field on touch devices.

Tested on my iPhone 6s on safari.

### PR DESCRIPTION
**Pull request by fritzmg**
Fixes #6382 

Same as #7583 for Contao 5.4+

### CODE REVIEW

**Review by aschempp — COMMENTED** [review]
Looks like we never unregister those event listeners? That could be a massive problem with Turbo...
Not sure if you wanna fix that in this PR as well?

**Comment by fritzmg** [comment]
I have at least added unregistering of the global document touchstart listener in this PR now.

**Comment by fritzmg** [comment]
> Looks like we never unregister those event listeners? That could be a massive problem with Turbo...
> Not sure if you wanna fix that in this PR as well?

I don't think we have to fix anything there. The eventlisteners are registered only once and only to the element itself. You don't have to remove eventlisteners on elements that will be removed from the DOM.

It's different for the global `touchstart` listener that I added - which is now fixed anyway.

File: core-bundle/assets/controllers/tooltip-controller.js [hunk 1]
```diff
@@ -12,6 +12,14 @@ export default class extends Controller {
         if (this.observer) {
             this.observer.disconnect();
         }
+
+        Object.entries(this.selectors).forEach(([selector, options]) => {
```
[the remark this answers is not in the document]
Comment (fritzmg) [reply core-bundle/assets/controllers/tooltip-controller.js]: Fixed in 4aef7c2199

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @fritzmg.

**Pull request merged**